server/channels/app/notification.go

```diff
@@ -879,6 +879,9 @@ func (a *App) SendNotifications(c request.CTX, post *model.Post, team *model.Tea
 				)
 			}
 		}
+		if user.IsRemote() && user.RemoteId != nil && *user.RemoteId != "" {
```

Comment:
`user.IsRemote()` already checks for `RemoteId`

```suggestion
		if user.IsRemote() {
```